Trappist/src/Promact.Trappist.Repository/TestSettings/TestSettingsRepository.cs

```diff
@@ -45,10 +37,28 @@ public string UpdateTestSettings(int id, Test testObject)
         /// </summary>
         /// <param name="id">The parameter "id" is used to get the Settings of a Test by its Id</param>
         /// <returns>Settings Saved for the selected Test</returns>
-        public Test GetTestSettings(int id)
+        public async Task<Test> GetTestSettingsAsync(int id)
+        {
+            string currentDate = DateTime.Now.ToString("MM/dd/yyyy HH:mm");
```

Comment:
the DateTime.Now has been converted to string to give the format which is to be matched with the client side.